components/script/dom/globalscope.rs

```diff
@@ -2179,42 +2179,42 @@ impl GlobalScope {
     pub fn add_uncaught_rejection(&self, rejection: HandleObject) {
         self.uncaught_rejections
             .borrow_mut()
-            .push(Heap::boxed(rejection.get()));
+            .push(*Heap::boxed(rejection.get()));
```

Comment:
This is creating a `Box<Heap<>>` and then immediately pulling it apart. I would either keep using `Box<...>` in the vector or figure out a way to create this `Heap` without putting it in a `Box<...>`.